components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1868,6 +1874,16 @@ where
             self.on_transfer_leader_msg(msg.get_message(), peer_disk_usage);
             Ok(())
         } else {
+            // This can be a message that sent when it's still a follower. Nevertheleast,
+            // it's meaningless to continue to handle the request as callbacks are cleared.
+            if msg.get_message().get_msg_type() == MessageType::MsgReadIndex
+                && self.fsm.peer.is_leader()
+                && (msg.get_message().get_from() == 0
```

Comment:
question: when get_from() == 0? is 0 a special meaning id?

Reply:
0 is the same as `self.id()`. raft-rs will treat it as if the request is issue locally.

Reply:
How about using `raft::INVALID_ID` instead.